pkg/validator/schema.go

```diff
@@ -251,6 +253,18 @@ func applySchemaCheck(conf *config.Configuration, checkID string, test schemaTes
 	} else if check.Target == config.TargetPod {
 		passes, issues, err = check.CheckPod(test.Resource.PodSpec)
 	} else if check.Target == config.TargetContainer {
+		containerIndex := funk.IndexOf(test.Resource.PodSpec.Containers, func(value corev1.Container) bool {
+			return value.Name == test.Container.Name
+		})
+		if test.Resource.Kind == "CronJob" {
+			prefix = "/spec/jobTemplate/spec/template/spec/containers/" + strconv.Itoa(containerIndex)
+		} else if test.Resource.Kind == "Pod" {
+			// Pod
+			prefix = "/spec/containers/" + strconv.Itoa(containerIndex)
+		} else {
```

Comment:
I think this should explicitly say `else if kind == Deployment || kind == DaemonSet ...`

The Kind here could hypothetically be any CRD. Maybe we need to just `log.Warning` in the `else`